### ISSUE TITLE
[MAS4.1.2] Value property is not properly defined for 'Quantity' text box of all card input data.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1346]
**Actual Result:**
When focus lands on the 'Quantity' text box, Narrator is reading 'Quantity editable text, min value 0 and max. value 0 Quantity editable text'. 
Value property is not defined properly for 'Quantity' text box.

**Note:** NVDA is reading 'Quantity spin button 0 Quantity' when focus lands on the 'Quantity' text box.

**Expected Result:**
Value property should be properly defined for “Quantity" text box.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Content will upload, Navigate to any "Show an Adaptive cards with all types of inputs " button by using Tab key and press Enter on it.
5. New data will upload.
6. Open Narrator (Ctrl + Win. key + Enter)
7. Navigate to 'Quantity text box' by using Tab key or press F12 to open developer tool and check.
8. Observe that Value property is not defined properly for 'Quantity' text box.

**User Impact:**
If Name, State, Value and Role property is not defined properly for controls then the users will get confuse about the purpose of that control.

**MAS Reference:**
[MAS4.1.2 - Name,Role,Value](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
1. [MAS4.1.2_Name for Quantity editable text box.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2579189/MAS4.1.2_Name.for.Quantity.editable.text.box.zip)
2. ![mas4 1 2_name value for quantity text box is not correct](https://user-images.githubusercontent.com/41668935/48460088-23e3c900-e7f3-11e8-9562-434d109359a9.png)

**Comment by compulim** [comment]
Investigations:
1. Write a simple HTML with `min` and `max`
1. Open in Edge and Narrator and see how it speak them out
- If it speak correctly, investigate Web Chat or Adaptive Cards
- If it doesn't speak correctly, this bug should belongs to Edge team

**Comment by corinagum** [comment]
Needs investigation

**Comment by corinagum** [comment]
@priysn 
I was able to confirm that when writing a simple index page (no Web Chat) and using the number input, Edge has the error where min and max are spoken as 0 by Narrator. This bug should be filed with Edge.

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
</head>
<body>
    <form>
        <input placeholder="Quantity" aria-label="Quantity" type="number" min="-5" max="5" tabindex="0">
    </form>
</body>
</html>
```

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
